src/sentry/migrations/0197_add_project_transaction_threshold.py

```diff
@@ -0,0 +1,121 @@
+# Generated by Django 1.11.29 on 2021-05-18 20:36
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+import sentry.db.models.fields.bounded
+import sentry.db.models.fields.foreignkey
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    # You'll also usually want to set this to `False` if you're writing a data
+    # migration, since we don't want the entire migration to run in one long-running
+    # transaction.
+    atomic = True
+
+    dependencies = [
+        ("sentry", "0196_add_restricted_member_limit"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="ProjectTransactionThreshold",
+            fields=[
+                (
+                    "id",
+                    sentry.db.models.fields.bounded.BoundedBigAutoField(
+                        primary_key=True, serialize=False
+                    ),
+                ),
+                ("threshold", models.IntegerField()),
+                ("metric", models.PositiveSmallIntegerField(default=1)),
+                ("date_updated", models.DateTimeField(auto_now=True)),
+                (
+                    "edited_by",
+                    sentry.db.models.fields.foreignkey.FlexibleForeignKey(
+                        null=True,
```

Comment:
Should this have `db_constraint=False` on it?

Reply:
Oh yes, I suppose it should -- updating

Reply:
I forgot to push my changes 😑 , added in 6d26588bb1749bd6b2961f11228e56c7b384d265